vhdbuilder/packer/install-dependencies.sh

```diff
@@ -75,9 +75,9 @@ cat << EOF >> ${VHD_LOGS_FILEPATH}
 EOF
 
 VNET_CNI_VERSIONS="
+1.1.3
 1.1.2
```

Comment:
can we remove 1.1.2? Our disk is almost full now so want to keep minimum stuff on it